substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -96,7 +96,11 @@ pub struct StorageParams {
 	#[arg(long, default_value_t = 1)]
 	pub warmups: u32,
 
-	/// The `StateVersion` to use. Substrate `--dev` should use `V1` and Polkadot `V0`.
+	/// The `StateVersion` to use.
+	///
+	/// Substrate `--dev` should use `V1` and Polkadot `V0`.
```

Comment:
```suggestion
	/// Substrate `--dev` should use `1` and Polkadot `0`.
```
I think its more understandable.

Reply:
Polkadot is now also at v1?